easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb

```
@@ -0,0 +1,25 @@
+easyblock = 'Bundle'
+
+name = 'Python-bundle'
+version = '3.10.4'
+
+homepage = 'https://easybuild.io/'
+description = """Python distribution with a number of widely used 
+extensions incl. NumPy, SciPy, Matplotlib, JupyterLab, MPI4PY, ... """
+
+local_gccver = '11.3.0'
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+dependencies = [
+    ('Python', version),
+    ('SciPy-bundle', '2022.05'),
+    ('matplotlib', '3.5.2'),
+    ('JupyterLab', '3.5.0', '', ('GCCcore', local_gccver)),
```

not needed, sub-toolchains are automatically mapped
```suggestion
    ('JupyterLab', '3.5.0'),
```

---

Done